src/api.py

```diff
@@ -86,7 +86,12 @@ def piechart_data():
 
 @app.route('/richestwallets', methods=['GET'])
 def richest_wallets():
-    wallets = analyzer.richest_wallets()
+    limit = request.args.get('limit', default='20')
+    try:
+        limit = int(limit, 10)
```

Comment:
base=10 is used by default, no need to pass it explicitly

Reply:
> base=10 is used by default, no need to pass it explicitly

removed ;)